apps/crn-server/test/data-providers/contentful/teams.data-provider.test.ts

```diff
@@ -251,97 +253,291 @@ describe('Teams data provider', () => {
+          linkedFrom: {
+            teamMembershipCollection: {
+              total: 1,
+              items: [
+                {
+                  role: 'Project Manager',
+                  inactiveSinceDate: null,
+                  linkedFrom: {
+                    usersCollection: {
+                      total: 1,
+                      items: [
+                        {
+                          ...getContentfulGraphqlTeamMembers(),
+                          labsCollection: {
+                            items: [null, { sys: { id: 'lab-2' } }],
+                          },
+                        },
+                      ],
+                    },
+                  },
+                },
+              ],
+            },
           },
-        ];
-        const team3 = getContentfulGraphqlTeam();
-        team3.toolsCollection!.items! = fullTools;
+        };
 
         contentfulGraphqlClientMock.request.mockResolvedValueOnce({
           teamsCollection: {
-            total: 3,
-            items: [team1, team2, team3],
+            total: 1,
+            items: [team],
           },
         });
 
-        const result = await teamDataProvider.fetch({
-          take: 8,
-          skip: 0,
+        const result = await teamDataProvider.fetch({});
+
+        expect(result).toEqual({
+          items: [expect.objectContaining({ labCount: 1 })],
+          total: 1,
+        });
+      });
+      test('should include only unique labs in the lab count', async () => {
+        const team = {
+          ...getContentfulGraphqlTeam(),
+          linkedFrom: {
+            teamMembershipCollection: {
+              total: 2,
+              items: [
+                {
+                  role: 'Key Personnel',
+                  inactiveSinceDate: null,
+                  linkedFrom: {
+                    usersCollection: {
+                      total: 1,
+                      items: [
+                        {
+                          ...getContentfulGraphqlTeamMembers(),
+                          labsCollection: {
+                            items: [
+                              { sys: { id: 'lab-1' } },
+                              { sys: { id: 'lab-2' } },
+                              { sys: { id: 'lab-3' } },
+                            ],
+                          },
+                        },
+                      ],
+                    },
+                  },
+                },
+                {
+                  role: 'Project Manager',
+                  inactiveSinceDate: null,
+                  linkedFrom: {
+                    usersCollection: {
+                      total: 1,
+                      items: [
+                        {
+                          ...getContentfulGraphqlTeamMembers(),
+                          labsCollection: {
+                            items: [
+                              { sys: { id: 'lab-1' } },
+                              { sys: { id: 'lab-2' } },
```

Comment:
I'd use a `lab-4` and a count of 4, that way you also test that the count does not just come from a single team membership